### PR TITLE
Corrected Module.md and version bumped.

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Make changes for feedback given by Pramodya

### CODE REVIEW

File: src/amazonsqs/Module.md [hunk 1]
```diff
@@ -56,19 +45,41 @@ Follow the method explained below to obtain AWS credentials.
...
+
+```ballerina
+amazonsqs:Configuration configuration = {
+    accessKey: "<ACCESS_KEY_ID>",
+    secretKey: "<SECRET_ACCESS_KEY>",
+    region: "<REGION>",
+    accountNumber: "<ACCOUNT_NUMBER>",
+    secureSocketConfig: {
+        trustStore: {
+            path: config:getAsString("<TRUSTSTORE_PATH>"),
```
Comment (pramodya1994) [inline src/amazonsqs/Module.md]:
```suggestion
            path: "<TRUSTSTORE_PATH>",
```

File: src/amazonsqs/Module.md [hunk 2]
```diff
@@ -56,19 +45,41 @@ Follow the method explained below to obtain AWS credentials.
...
+            password: config:getAsString("<TRUSTSTORE_PASSWORD>")
```
Comment (pramodya1994) [inline src/amazonsqs/Module.md]:
```suggestion
            password: <TRUSTSTORE_PASSWORD>"
```

File: src/amazonsqs/tests/main_test.bal [hunk 3]
```diff
@@ -24,6 +24,15 @@ Configuration configuration = {
     secretKey: config:getAsString("SECRET_ACCESS_KEY"),
     region: config:getAsString("REGION"),
     accountNumber: config:getAsString("ACCOUNT_NUMBER")
+
```
Comment (pramodya1994) [inline src/amazonsqs/tests/main_test.bal]: shall we remove these comented lines?

Comment (Maninda) [reply src/amazonsqs/tests/main_test.bal]: Removed

File: src/amazonsqs/Module.md (hunk 2 above)
Comment (Maninda) [reply src/amazonsqs/Module.md]: Fixed

File: src/amazonsqs/Module.md (hunk 1 above)
Comment (Maninda) [reply src/amazonsqs/Module.md]: Fixed

**Pull request merged**